src/aks-preview/azext_aks_preview/tests/latest/test_agentpool_decorator.py

```diff
@@ -1119,6 +1154,9 @@ def test_get_enable_vtpm(self):
     def common_get_enable_fips_image(self):
         self.common_get_enable_fips_image()
 
+    def common_get_enable_kata_image(self):
```

Comment:
need to rename this to `test_get_enable_kata_image`, same for the above occurrence. (BTW, common_get_enable_fips_image is a bad example, could you help fix it by changing it to `test_get_enable_fips_image` as well?)